modules/TargetSafety.py

```diff
@@ -308,6 +334,78 @@ def process_toxcast(toxcast: str) -> DataFrame:
         ).alias("study"),
     )
 
+def process_pharmacogenetics(spark: SparkSession, pgx_df: DataFrame, cache_dir: str) -> DataFrame:
+    """Given the pharmacogenetics evidence, extract the evidence related to target toxicity."""
+    evidence_unique_cols = [
```

Comment:
This variable not used.

Reply:
solved in https://github.com/opentargets/evidence_datasource_parsers/pull/185/commits/d6b7975f178820438e8a37dc83638d022e3656bf